packages/world/src/ResourceSelector.sol

```diff
@@ -44,7 +49,26 @@ library ResourceSelector {
    * Convert a selector to a string for more readable logs
    */
   function toString(bytes32 resourceSelector) internal pure returns (string memory) {
-    return string(abi.encodePacked(getNamespace(resourceSelector), "/", getFile(resourceSelector)));
+    bytes16 namespace = getNamespace(resourceSelector);
+    bytes16 file = getFile(resourceSelector);
+    return
+      string(
+        abi.encodePacked(
+          namespace == ROOT_NAMESPACE ? ROOT_NAMESPACE_STRING : namespace,
+          "/",
```

Comment:
do we have any guard rails to keep folks from using `/` in their namespace or filename and clobbering these "paths"?

Reply:
similar to the q above: we don't have checks that prevent folks from having `/` in their namespace or file name on-chain right now, but since there is no security concern + this is only used for logging / error messages + the CLI prevents the use of special characters in namespace/file name, I think again not worth the gas cost of preventing this on-chain. The only one who gets hurt by using a `/` is the person using it (because their logs might be harder to read), and it requires explicit effort to go around the CLI and use a `/`